apis/config/v1beta1/configuration_types.go

```diff
@@ -449,6 +449,11 @@ type Resources struct {
 	// Transformations defines how to transform PodSpec resources into Workload resource requests.
 	// This is intended to be a map with Input as the key (enforced by validation code)
 	Transformations []ResourceTransformation `json:"transformations,omitempty"`
+
+	// DeviceClassMappings defines mappings from device classes to logical resources
+	// for Dynamic Resource Allocation support.
+	// +optional
+	DeviceClassMappings []DeviceClassMapping `json:"deviceClassMappings,omitempty"`
```

Comment:
I notice we are not fetching deviceClasses.

There is a potential issue here where someone can specify a deviceClassMapping that doesn't exist. 

Should we fetch deviceClasses to make sure they exist before we count them?